.gitignore

```diff
@@ -1 +1,4 @@
 .idea
+*.iml
+*.ipr
```

Comment:
For completeness I'd add `*.iws`.

Reply:
Nice catch! Good point, forgot that one!! Just updated, going to resolve this comment thread :) Thanks @baron1405 !

By the way, have you seen https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore ? :) 

I stumbled across that over break and my mind slightly 🤯 - they have a ton for different languages and what have you; pretty neat, just thought I'd share :)